### PR TITLE
Fix broken IPFS icons

### PR DESCRIPTION
**Pull request by hyphenized**
Before:

After:

## To Test: 
- [ ] Load the test wallet
- [ ] Check icons display correctly

Latest build: [extension-builds-2702](https://github.com/tallycash/extension/suites/9627957205/artifacts/459926057) (as of Fri, 02 Dec 2022 09:16:24 GMT).

### CODE REVIEW

**Review by jagodarybacka — COMMENTED** [review]
It's spamming with errors on opening tokens selector, can we do something about it?
image

**Comment by jagodarybacka** [comment]
Seems like our IntersectionObserver needs some love to work in tests 😅 btw I made it a hook [here](https://github.com/tallycash/extension/pull/2703/files#diff-48e41e2a2f75ebf6ef1da526ecb3132feafd847646c10b353970af8c0fac981a), I will update this instance if your PR will be merged first :)

File: ui/components/Shared/SharedAssetIcon.tsx [hunk 1]
```diff
@@ -1,35 +1,54 @@
...
+  useEffect(() => {
+    const isIpfsURL = /^ipfs:/.test(logoURL)
+
+    if (isIpfsURL) {
+      fetch(httpURL).then(async (response) => {
+        if (
+          response.ok &&
+          response.headers.get("content-type") === "text/html"
+        ) {
+          const base = "data:image/svg+xml;base64,"
```
[the remark this answers is not in the document]
Comment (hyphenized) [reply ui/components/Shared/SharedAssetIcon.tsx]: Changed to `imageURL` 👀

**Review by jagodarybacka — APPROVED** [review]
Works nice ❤️

**Pull request merged**